src/libstd/sys/unix/stack_overflow.rs

```diff
@@ -55,8 +57,10 @@ mod imp {
 
 
     // This is initialized in init() and only read from after
+    #[cfg(not(test))]
```

Comment:
Hm instead of sprinkling these around, perhaps this could be at the top with something like `#![cfg_attr(test, allow(dead_code))]`?